packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardUtils.ts

```diff
@@ -0,0 +1,146 @@
+import { Query } from '@sage-bionetworks/synapse-types'
+import dayjs from 'dayjs'
+import { formatDate } from '../../utils/functions/DateFormatter'
+import { ReleaseCardSchema } from './ReleaseCard'
+import {
+  ButtonToExplorePageConfig,
+  ReleaseCardStat,
+  StatConfig,
+} from './ReleaseCardTypes'
+
+const MISSING_STAT_DEFAULT_VALUE = 'Unknown'
+
+const formatNumber = (num: number) => {
+  if (num < 1_000_000) return num.toLocaleString()
+  const numberFormatter = new Intl.NumberFormat([], {
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 2,
+    notation: 'compact',
+    localeMatcher: 'best fit',
+  })
+  return numberFormatter.format(num)
+}
```

Comment:
When a statistic is a very large number, use [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) to display a shortened version of the number, e.g. 1.234T instead of 123,456,789,012,345.